lib/matplotlib/axes/_base.py

```diff
@@ -3902,18 +3902,18 @@ def _make_twin_axes(self, *kl, **kwargs):
 
     def twinx(self):
         """
-        Create a twin Axes sharing the xaxis
+        Create a twin Axes instance sharing the x-axis with self
```

Comment:
not clear what self is here? (Basically not sure this is any clearer than the original wording)